scss/detail.scss

```diff
@@ -0,0 +1,13 @@
+h1 {
```

Comment:
We try as specific as possible when targeting styles, so use the class names you already declared. This is because if later on the future you want to add another h1 that is not part of your book component, it would has this same style even if you dont intent to, so apply styles as specific as possible.